sympy/physics/mechanics/_actuator.py

```diff
@@ -25,15 +25,29 @@
 from sympy.physics.mechanics._pathway import PathwayBase
 
 if USE_SYMENGINE:
-    from sympy.core.backend import Basic as ExprType
+    from sympy.core.backend import (
+        Basic as ExprType,
+        sympify as sympify_symengine,
+    )
+    from sympy.core.sympify import sympify as sympify_sympy
+
+    def sympify(*args, **kwargs):
+        if kwargs.get('strict') is True:
+            return sympify_symengine(sympify_sympy(*args, **kwargs))
```

Comment:
Are there cases where this double `sympify` is actually useful? Af far as I've notices the `symengine.sympify` is relatively "strict".

Reply:
Yes, simplest example using the above function is `sympify('k', strict=True)`. 

```python
# export USE_SYMENGINE=0
>>> from sympy.core.backend import sympify
>>> sympify('x', strict=True)
Traceback (most recent call last):
    ...
    raise SympifyError(a)
sympy.core.sympify.SympifyError: SympifyError: 'k'
```

```python
# export USE_SYMENGINE=1
>>> from sympy.core.backend import sympify
>>> x = sympify('x')
>>> type(x)
<class 'symengine.lib.symengine_wrapper.Symbol'>
```

Reply:
Would it not be better to handle this in `sympy.core.backend` rather than here in the physics module?

Reply:
I guess actually this version could be a lot slower than symengine's sympify. I would suggest something like:
```python
def sympify(obj, *, strict=False):
    if strict and isinstance(obj, str):
        raise
    return sympify_symengine(obj)
```

Reply:
Thanks for the input. Addressed in https://github.com/sympy/sympy/pull/25116/commits/09c935c698182264eec539af7716106868ea9b72.